sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -70,7 +69,6 @@ public boolean isConnectionEndpointRediscoveryEnabled() {
      *
      * @return the {@linkplain DirectConnectionConfig}.
      */
-    @Beta(value = Beta.SinceVersion.V4_8_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
     public DirectConnectionConfig setConnectionEndpointRediscoveryEnabled(boolean connectionEndpointRediscoveryEnabled) {
```

Comment:
```suggestion
    public DirectConnectionConfig enableConnectionEndpointRediscovery() {
```
If the user calls this API, then we set it to `true`, else, it is `false` by default. 
@moderakh @FabianMeiswinkel - to be consistent with other APIs on public surface, I think we should rename this API too, thoughts ?
@xinlian12  and I agree on the above suggestion.

Reply:
@kushagraThapar all the apis in DirectConnectionConfig has the "set" prefix.

I think we should keep it as `setConnectionEndpointRediscoveryEnabled` to be consistent with apis in `DirectConnectionConfig`

The other thing we need to think about is if in future we decide to change the default feature and enable this by default if we go with `enableConnectionEndpointRediscovery()` won't support disabling, we will have to add another api for disabling.

Reply:
That's a very good point, what about -> 
`enableConnectionEndpointRediscovery(boolean connectionEndpointRediscoveryEnabled)`

Reply:
@moderakh Regarding the other APIs in `DirectConnectionConfig` -> there are no Boolean APIs as of now.. So this is the first one which is going out.. Hence we think, we should rename it to above.

Reply:
Checked on other boolean APIs -> like in `CosmosQueryRequestOptions`, we have other boolean APIs with same `set` prefix.